src/fragments/lib/graphqlapi/graphql-from-node.mdx

```diff
@@ -78,6 +78,7 @@ For functions written using CommonJS, you will need to install version 2 of `nod
   }
 }
 ```
+NOTE: The examples bellow are using ES module syntax and currently not supported by amplify mock function. If you wish to use amplify mock please use commonjs syntax with node-fetch v2.
```

Comment:
```suggestion
```

nit: this is now noted above https://github.com/aws-amplify/docs/pull/4441/files#diff-47ceb664ff5a7a62e1a2d7539cd4e2b6d7581fb7510fd5c1a6f76648a398204aR45

Reply:
@josefaidt Noted and commited :)